Funcky.Analyzer/Funcky.Analyzer.Package/tools/install.ps1

```diff
@@ -0,0 +1,250 @@
+param($installPath, $toolsPath, $package, $project)
```

Comment:
This is needed for packages.config, right? Do we want to support that?

Reply:
No, clue, autogenerated from template.

Reply:
Hmm that's odd. NuGet actively [discourages](https://github.com/NuGet/Home/issues/3656) the use of these scripts, but most of the analyzers from microsoft still include the script...

Reply:
The Microsoft Documentation of March 2020 here [How to write a Roslyn Analyzer](https://devblogs.microsoft.com/dotnet/how-to-write-a-roslyn-analyzer/) clearly states to install the "Visual Studio extension development" workload in VS2019. If you follow the steps, you should use the "Analyzer with Code Fix (.NET Standard)" which creates both a install and an uninstall script. 

I don't see why I should delete arbitray files in a template from MS without a good understanding on why it is done this way.